CMakeLists.txt

```diff
@@ -381,9 +374,9 @@ endif()
 
 #-------------------------------------------------------------------------------
 # Compiler Target Options
-# By default, all compiler targets supported by the current platform are
-# enabled. Some compiler targets like CUDA will install external deps as
-# needed at configure time. This can be changed with:
+# By default, all compiler targets supported by the current platform and that don't
+# need a SDK are enabled. Some compiler targets like CUDA will install external
+# deps as needed at configure time. This can be changed with:
```

Comment:
Should update the website docs to match this code: https://iree.dev/building-from-source/getting-started/#optional-components

Source:
https://github.com/iree-org/iree/blob/10877f61c2e73d578bb889012b84bbd1edf15ed0/docs/website/docs/building-from-source/getting-started.md?plain=1#L169-L183

Reply:
Yes thanks! The documentation changes where one of the reasons this was still in draft :)